gigaword-indexer/src/main/java/edu/isi/vista/gigawordIndexer/ConcatenatedAnnotatedGigawordDocuments.java

```diff
@@ -0,0 +1,133 @@
+package edu.isi.vista.gigawordIndexer;
+
+import com.google.common.collect.AbstractIterator;
+import de.tudarmstadt.ukp.dkpro.core.io.penntree.PennTreeUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.io.BufferedReader;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * The LDC distributes Gigaword as a moderate number of gzipped files, each of which has many documents
+ * concatenated together.  This class lets you iterate over the documents stored in such a file.
+ */
+
+public class ConcatenatedAnnotatedGigawordDocuments implements Iterable<ConcatenatedGigawordDocuments.Article> {
```

Comment:
In the Javadoc, can you mention that:

* this class was authored by you all ("Technische Universität Darmstadt" or whatever the preferred nomenclature is)
* it does not preserve the offsets from the original text, so it shouldn't be used for the curated training process.
* it is included here for Darmstadt's convenience.

Reply:
Fixed in newest commit.